5-appinfra/modules/cicd-pipeline/locals.tf.sav

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
```

Comment:
@yliaog looks like there are duplicate files in the `cicd-pipeline/` folder - can you reconcile the `.sav` and `.bak` files?

Reply:
sorry, added them by mistake, removed  now.